src/test/java/api/support/RestAssuredClient.java

```diff
@@ -43,7 +44,7 @@ public static RestAssuredClient defaultRestAssuredClient() {
 
   public RequestSpecification beginRequest(String requestId) {
     return given()
-      .log().all()
+      .log().ifValidationFails()
```

Comment:
What validation does this refer to?

Reply:
Status code of response I think.

Reply:
> Status code of response I think.

As I understand it, only some of the RestAssured request set expectations about what an acceptable status code is. What happens for the requests that do not specify that?

Reply:
For this case I use `log().ifStatusCodeMatches(greaterThan(300))` - response is logged when status code > 300